staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -828,8 +840,20 @@ func (meal *cfgMeal) finishQueueSetReconfigsLocked() {
 		if limited := plState.pl.Spec.Limited; limited != nil {
 			if qCfg := limited.LimitResponse.Queuing; qCfg != nil {
 				meal.maxWaitingRequests += int(qCfg.Queues * qCfg.QueueLengthLimit)
+
+				// Max seats allocatable from work estimator is MAX(1, MIN(0.15 * nominalCL, nominalCL/handSize)).
+				// This is to keep max seats relative to total available concurrency with a minimum value of 1.
+				// 15% of nominal concurrency was chosen since it preserved the default max seats for apiserver's default
+				// total server concurrency of 600 (--max-requests-inflight=400, --max-mutating-requests-inflight=200).
+				// This ensures most clusters with relatively high inflight requests will continue to use a max seats of 10
```

Comment:
It is odd to mention total server capacity here but not relative shares when explaining where the value 10 comes from, since relative shares is a critical part of that calculation.

Reply:
> It is odd to mention total server capacity here but not relative shares when explaining where the value 10 comes from, since relative shares is a critical part of that calculation. 

I clarified this to say that 15% if based on preserving 10 when using the default priority level configurations with the default total server concurrency.